types/node/ts4.0/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+// TODO: duplicate or extend?
```

Comment:
I copied this from `types/node/tsconfig.json`. I wasn't sure whether to duplicate or use `extends`?